core/wiki/macros/translink.tid

```diff
@@ -3,14 +3,25 @@ tags: $:/tags/Macro
 
 \define translink(title,mode:"block")
 \whitespace trim
-<div style="border:1px solid #ccc; padding: 0.5em; background: black; foreground; white;">
+<div class={{{ [[tc-translink]] [[$title$]is[missing]then[tc-translink-inline]] [[$mode$]match[block]then[tc-translink-block]] [[$mode$]match[inline]then[tc-translink-inline]] +[first[2]] +[join[ ]] }}}>
 <$link to="""$title$""">
-<$text text="""$title$"""/>
+<h1 style="font-weight: bold; font-size: 1.2em;"><$text text="""$title$"""/></h1>
 </$link>
-<div style="border:1px solid #ccc; padding: 0.5em; background: white; foreground; black;">
 <$transclude tiddler="""$title$""" mode="$mode$">
 "<$text text="""$title$"""/>" is missing
 </$transclude>
 </div>
-</div>
+<style>
```

Comment:
The core doesn’t use stylesheets embedded in HTML like this. One problem with this approach is that if there are many repetitions of this macro displayed at once then the browser will get a bunch of redundant stylesheets.

The core macros that need classes can define them in the base.tid stylesheet in the Vanilla theme.